Dockerfile

```diff
@@ -153,6 +154,23 @@ RUN set -ex \
  && pipenv install --dev --skip-lock --clear \
  && apk del --no-network .build-deps
 
+# Build a static version of the front end to serve
+COPY ./package*.json ./
+
+RUN set -ex \
+ && apk add --no-cache --virtual .node-build-deps \
+    make \
+    build-base \
+    python2 \
```

Comment:
It's a build requirement of `node-gyp` at whatever version we're using (or whatever version one of our dependencies is using, it may just get built for `node-sass`). We should really update those front end dependencies.